cmd/bad-key-revoker/main.go

```diff
@@ -76,8 +78,11 @@ func (bkr *badKeyRevoker) countUncheckedKeys(ctx context.Context) (int, error) {
 		&count,
 		`SELECT COUNT(*)
 		FROM (SELECT 1 FROM blockedKeys
-		WHERE extantCertificatesChecked = false
+		WHERE extantCertificatesChecked = 0
+		OR (extantCertificatesChecked < 2 AND added BETWEEN NOW() - INTERVAL ? SECOND AND NOW() - INTERVAL ? SECOND)
```

Comment:
I think this can just be:

```suggestion
		WHERE extantCertificatesChecked = 0
		OR (extantCertificatesChecked = 1
			AND added BETWEEN NOW() - INTERVAL ? SECOND
			AND NOW() - INTERVAL ? SECOND)
```

The case where `extantCertificatesChecked = 0` is already covered by your first clause, so the only value `< 2` can match on is `1`.

Reply:
`extantCertificatesChecked` is (weirdly) signed; since it *theoretically* could be a negative value, I figured I'd write the check this way. Do you think there's a strong reason to use `= 1` instead?

Reply:
I think that's perfectly reasonable and in the event that it ended up negative this would still work, albeit with more cycles. Where as with `= 1` it would not.